panel/browser-panel.cpp

```diff
@@ -253,9 +256,8 @@ void QCefWidgetInternal::Init()
 			CefRefPtr<CefDictionaryValue>(),
 #endif
 			rqc);
-#ifdef _WIN32
+
```

Comment:
Probably don't need this whitespace addition, unless the visual separation is truly needed.